### ISSUE TITLE
Background scan not working

### ISSUE THREAD

**Issue by webworxshop** [vitas/beaconloc#14]
Hi,

I'm testing this out with the PyBeacon software (https://github.com/nirmankarta/PyBeacon) on my laptop and my HTC M7 (Android version 5.0.2). I can see the beacon just fine in a manual scan, but the background scan never seems to detect it and as such my actions aren't executed.

Any ideas? Happy to collect any debug logs, etc. you need.

**Comment by vitas** [comment]
the background scan and an action triggering only work when the app in the background (UI is not open).

**Comment by webworxshop** [comment]
Yeah, I read that in the wiki. I already tried it with the app backgrounded and it still doesn't work.

**Comment by markrennie71** [comment]
I am having the same problem. I close the app and background scanning doesn't work. Am I missing something as I want to replace owntracks with this to detect my beacons
